crates/wash-cli/src/dev.rs

```diff
@@ -91,6 +92,11 @@ pub struct DevCommand {
     #[clap(name = "code-dir", long = "work-dir", env = "WASH_DEV_CODE_DIR")]
     pub code_dir: Option<PathBuf>,
 
+    /// Directories to ignore when watching for changes. This should be set
+    /// to directories where generated files are placed, such as `target/` or `dist/`
```

Comment:
Should the first sentence be separated from the second? And maybe an indication it can be specified multiple times

Reply:
Good idea, I'll incorporate this in #3140 just to keep merging simple!